syntax = "proto2";
package DB.Protos;

import "cnch_common.proto";
import "data_models.proto";

message VersionedPartition {
    required StorageID storage_id = 1;
    required bytes partition = 2;
    required uint64 last_update_time = 3;
}

message VersionedPartitions {
    required StorageID storage_id = 1;
    repeated VersionedPartition versioned_partition = 2;
}
